src/mumble/GlobalShortcut_win.cpp

```diff
@@ -3,12 +3,20 @@
 // that can be found in the LICENSE file at the root of the
 // Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
+// For detailed info about RAWKEYBOARD handling:
+// https://blog.molecular-matters.com/2011/09/05/properly-handling-keyboard-input
+
 #include "GlobalShortcut_win.h"
 
 #include "MainWindow.h"
+#include "OverlayClient.h"
```

Comment:
How is the overlay related to the new shortcut system?

Reply:
It's not, I had the include in the file for some reason and forgot to remove it.